test/utils/skipLogic.js

```diff
@@ -320,30 +326,23 @@ tap.test('.skipLogic()', { autoend: true }, (t) => {
       }]
     }
 
-    skipLogicGroupCheck(scope, 'yes', check)
+    skipLogicGroupCheck(scope, check)
     t.notOk(scope.field.show)
 
     check.logicGate = 'or'
-    skipLogicGroupCheck(scope, 'yes', check)
```

Comment:
you would just remove the check after the var initialisations and that should still have these tests pass